modal_proto/api.proto

```diff
@@ -2909,6 +2909,11 @@ message VolumeCopyFilesRequest {
   bool recursive = 4;
 }
 
+message VolumeDelete2Request {
```

Comment:
Don't think this and `VolumeRenameRequest` should be duplicated for v2 (will divulge details/discuss out of band)